help.go

```diff
@@ -366,3 +368,53 @@ func checkCommandCompletions(c *Context, name string) bool {
 	ShowCommandCompletions(c, name)
 	return true
 }
+
+func Wrap(input string, offset int, wrapAt int) string {
+	var sb strings.Builder
+
+	lines := strings.Split(input, "\n")
+
+	for i, line := range lines {
+		if i != 0 {
+			sb.WriteString(strings.Repeat(" ", offset))
+		}
+
+		sb.WriteString(wrapLine(line, offset, wrapAt))
+
+		if i != len(lines)-1 {
+			sb.WriteString("\n")
+		}
+	}
+
+	return sb.String()
+}
+
+func wrapLine(input string, offset int, wrapAt int) string {
+	if wrapAt <= offset || len(input) <= wrapAt-offset {
+		return input
+	}
+
+	lineWidth := wrapAt - offset
+	words := strings.Fields(strings.TrimSpace(input))
+	if len(words) == 0 {
+		return input
+	}
+
+	wrapped := words[0]
+	spaceLeft := lineWidth - len(wrapped)
+	for _, word := range words[1:] {
+		if len(word)+1 > spaceLeft {
+			wrapped += "\n" + strings.Repeat(" ", offset) + word
+			spaceLeft = lineWidth - len(word)
+		} else {
+			wrapped += " " + word
+			spaceLeft -= 1 + len(word)
+		}
+	}
+
+	return wrapped
+}
+
+func offset(input string, fixed int) int {
```

Comment:
Probably not a big deal, but this function name is shadowed a few times in the other functions being introduced.

Reply:
If I understand correctly it should be safe, because there's only one `offset` function, and the other `offset`'s are `int` variables, but maybe it could be made less confusing. I could rename all the `offset` variables to `off` if you prefer?

Reply:
The current iteration doesn't seem to cause any problems, and doesn't seem to set off any linters I know of.

I think it should be alright as is, and renaming `offset` should be fine as well. Up to you.